tests/integration/converters/test_ome_tiff.py

```diff
@@ -44,6 +45,49 @@ def test_ome_tiff_converter(tmp_path, open_fileobj, preserve_axes):
             np.testing.assert_array_equal(region_data, level_data)
 
 
+@pytest.mark.parametrize(
+    "compressor",
+    [
+        tiledb.WebpFilter(WebpInputFormat.WEBP_RGB, quality=0, lossless=False),
+        tiledb.WebpFilter(WebpInputFormat.WEBP_RGB, quality=50, lossless=False),
+        tiledb.WebpFilter(WebpInputFormat.WEBP_RGB, quality=100, lossless=False),
+        tiledb.WebpFilter(WebpInputFormat.WEBP_RGB, quality=100, lossless=True),
+    ],
+)
+def test_ome_tiff_comverter_webp_rgb_compressor(tmp_path, compressor):
+    input_path = str(get_path("CMU-1-Small-Region.ome.tiff"))
+    output_path = str(tmp_path)
+    OMETiffConverter.to_tiledb(input_path, output_path, compressor=compressor)
+
+    with TileDBOpenSlide.from_group_uri(output_path) as t:
+        assert len(tiledb.Group(output_path)) == t.level_count == 2
+
+        schemas = (
+            get_schema(2220, 2967, compressor=compressor),
+            get_schema(574, 768, compressor=compressor),
+        )
+        shape = schemas[0].shape[::-1]
+        assert t.dimensions == (shape[0] // 3, shape[1])
+
+        for i in range(t.level_count):
+            shape = schemas[i].shape[::-1]
+            assert t.level_dimensions[i] == (shape[0] // 3, shape[1])
+            with tiledb.open(str(tmp_path / f"l_{i}.tdb")) as A:
+                assert A.schema == schemas[i]
+
+        region = t.read_region(level=0, location=(100, 100), size=(300, 400))
+        assert isinstance(region, np.ndarray)
+        assert region.ndim == 3
+        assert region.dtype == np.uint8
+        img = PIL.Image.fromarray(region)
+        assert img.size == (300, 400)
+
+        for level in range(t.level_count):
+            region_data = t.read_region((0, 0), level, t.level_dimensions[level])
+            level_data = t.read_level(level)
+            np.testing.assert_array_equal(region_data, level_data)
```

Comment:
For lossy compression options this test will always fail. Even using `numpy.testing.assert_allclose` can fail in case the image contains sharp variations.

Reply:
Addressed at https://github.com/TileDB-Inc/TileDB-BioImaging/pull/65/commits/3b06648abdf2df716ebc9ed7ff70e8b49ab49aed using `skimage.metrics.structural_similarity` that you introduced at https://github.com/TileDB-Inc/TileDB-BioImaging/pull/61.